Adjust readme instructions to version 3 (#599)
ezsystems/ezplatform: README.md

Assuming you have prerequisites sorted out, you can get the install up and running with the following commands in your terminal:

``` bash
composer create-project --keep-vcs ezsystems/ezplatform ezplatform ^3
cd ezplatform
```

**Note:** If composer is installed locally instead of globally, the first command will start with `php composer.phar`.

> You will need to fill your database connection credentials (hostname, login, password) into environment file.  

Reviewer: ```suggestion
You must add your database connection credentials (hostname, login, password) to the environment file.  
```

> To do so, change parameters prefixed with `DATABASE_` accordingly in `.env` file placed in main project directory.

Reviewer: ```suggestion
To do this, in the main project directory, the `.env` file, change the parameters that are prefixed with `DATABASE_` as necessary.
```

> It is *recommended* to store the database credentials in your .env.local file and not commit it to the Version Control System.

Reviewer: ```suggestion
Store the database credentials in your `.env.local` file. Do not commit the file to the Version Control System.
```

> To install eZ Platform (insert base data into database) use:

Reviewer: ```suggestion
Use the following command to install eZ Platform (insert base data into the database):
```
